src/history.jl

```diff
@@ -256,8 +256,8 @@ nrests(ch::RestartedHistory) = Int(ceil(ch.iters/ch.restart))
 Determine whether a collection `x` is plotable. Only vectors and matrices are
 such objects.
 """
-plotable{T<:Real}(::VecOrMat{T})::Bool = true
```

Comment:
Just asked and it seems to be a bug when using the "assignment form" of function definitions. I agree that it should really matter here.